optaplanner-core/src/main/java/org/optaplanner/core/config/constructionheuristic/placer/QueuedValuePlacerConfig.java

```diff
@@ -87,6 +87,7 @@ public void setMoveSelectorConfig(MoveSelectorConfig moveSelectorConfig) {
     // Builder methods
     // ************************************************************************
 
+    @SuppressWarnings("deprecation")
```

Comment:
Can we add a comment behind it "for what"? Mainly as an expiration date: when we can remove this line.
For example:

@SuppressWarnings("deprecation") // Remove in 8.0 when it doesn't call  foo.bar() any more